### PR TITLE
LPS-143989 Create PATCH UserGroup by ExternalReferenceCode for User Group Headless API

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserGroupResourceImpl.java [hunk 1]
```diff
@@ -225,6 +225,12 @@ private DTOConverterContext _getDTOConverterContext(long userGroupId) {
 					ActionKeys.UPDATE, userGroupId,
 					"putUserGroupByExternalReferenceCode",
 					_userGroupModelResourcePermission)
+			).put(
+				"update-by-external-reference-code",
```
Comment (drewbrokke) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserGroupResourceImpl.java:229]: Above, you used "patch" as the action, but used "update" here. For consistency, let's use "patch" in both places.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one comment about action naming, otherwise this looks good.

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:reevaluate:1377401_201

**Comment by drewbrokke** [comment]
ci:reopen

**Comment by erickmonteiroo** [comment]
ci:forward

**Comment by epark18** [comment]
Pushed to BChan.
[brianchandotcom#113184](https://github.com/brianchandotcom/liferay-portal/pull/113184)

**Pull request closed without merging**